src/translator/service.h

```diff
@@ -91,8 +91,9 @@ class Service {
   /// @param [in] responseOptions: Options indicating whether or not to include
   /// some member in the Response, also specify any additional configurable
   /// parameters.
-  std::future<Response> translate(std::string &&source,
-                                  ResponseOptions options = ResponseOptions());
+  void translate(std::string &&source, 
```

Comment:
Callback not in the documentation. 

Also, translate has a callback, but translateMultiple doesn't?

Reply:
I'll break extension and bindings if I change translateMultiple (translateMultiple is aliased as translate for extension at bindings). I have not heard from extension for a while regarding their async strategy to be able to modify this into callback. 

/cc @abhi-agg @motin

Reply:
We could provide a thin wrapper for them probably, but it's not ideal. Let's hear from mozilla ppl. I'd rather not put a temporary hack that would become permanent later.

Reply:
Fixed documentation, leaving open until extension resolves the other half.